README.md

```diff
@@ -80,8 +82,9 @@ graph LR
     API-.->|"Read</br>[psycopg]"|Database
     API-.->|"Read</br>[JSON/HTTPS]"|CFFDRS_API
     API-.->|"Uses</br>[Reads from disk]"|Files
-    API-. "Fetch fire weather data</br>[JSON/HTTPS]" .->wf1
+    API-. "Fetch fire weather data</br>[JSON/HTTPS]" .->wso2
```

Comment:
```suggestion
    API-. "Fetch fire weather data</br>[JSON/HTTPS]" .->ws02
```

Reply:
Vendor says it's "O": https://wso2.com/api-manager/